pkg/skaffold/build/tag/git_commit_test.go

```diff
@@ -458,30 +457,30 @@ func TestGitCommitSubDirectory(t *testing.T) {
 		gitInit(t.T, tmpDir.Root()).mkdir("sub/sub").commit("initial")
 		workspace := tmpDir.Path("sub/sub")
 
-		tagger, err := NewGitCommit("", "Tags")
+		tagger, err := NewGitCommit("", "Tags", "")
```

Comment:
can you add test cases that use the exclude functionality also, basically duplicating each test?

Reply:
it doesn't make sense for every test, but i added it where appropriate